documentacion/onepay/README.md

```diff
@@ -226,7 +226,36 @@ cart.add(Item(description="Zapatos",
               additional_data=None, expire=None))
 ```
 
-El monto en el carro de compras debe ser positivo, en caso contrario se lanzará una excepción.
+Se permite agregar un `Item` al carro con amount negativo que generalmente representa un descuento. Lo importante es que el monto total en el carro de compras debe ser positivo, en caso contrario se lanzará una excepción.
```

Comment:
I’d put `amount` in backticks as it references to the parameter name in code. But it’s probably just a personal preference. If you like it better without formatting that word as code it’s ok too